manager-bundle/src/Command/MaintenanceModeCommand.php

```diff
@@ -43,26 +43,62 @@ protected function configure(): void
     {
         $this
             ->setName('contao:maintenance-mode')
-            ->addArgument('state', InputArgument::REQUIRED, 'Use "enable" to enable and "disable" to disable the maintenance mode. If the state is already the desired one, nothing happens. You can also use "on" and "off".')
+            ->setAliases(['lexik:maintenance:lock', 'lexik:maintenance:unlock'])
```

Comment:
> We still need to add the aliases though for BC 😉

Besides that there is no BC promise for command names, why do you want to provide a BC layer here? In case someone has added `vendor/bin/contao-console lexik:maintenance:lock` to their deploy script?

Reply:
exactly, deploy scripts or any other automation tools or CLI aliases.

Reply:
Not sure this is a good idea. People like @richardhj are testing on command existance to see which version is used. Something along the lines of

- list all commands
- if it contains `lexik:maintenance` then do this
- otherwise this

with an alias, it might possibly break.

Also, if you're updating the version, you are also updating the deployment, so you probably have to adjust deployment logic anyway.

Reply:
the point of BC is that you don't need to adjust, but you do get a deprecation warning 🤷

Reply:
🤷 I won't be affected by either change. I understand both positions. But having a lexik: command while having a conflict on the lexik bundle feels wrong.